extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -517,7 +495,18 @@ private function doTableCleanup( DatabaseBase $db, $table, $city_id, $wiki_id_co
 	}
 
 	private function log( string $message ) {
-		WikiaLogger::instance()->info( $message );
+		$this->info( $message );
+	}
+
+	/**
+	 * @param string $msg
+	 * @param array $context
+	 */
+	private function info( string $msg, array $context = [] ) {
+		// send logs to stdout as well
+		Wikia::log( __CLASS__, false, $msg, true, true );
```

Comment:
Let's use `$this->output` instead

Reply:
This script does not inherit from `Maintenance` class.